dhall-openapi/README.md

```diff
@@ -6,6 +6,59 @@ For installation or development instructions, see:
 
 ## Introduction
 
-This `dhall-openapi` package provides an OpenAPI to Dhall compiler.
+This `dhall-openapi` package provides an OpenAPI 2.0 (a.k.a. Swagger) to Dhall
+compiler.
```

Comment:
I think this might be a bit easier to parse:

```suggestion
This `dhall-openapi` package provides a compiler from OpenAPI 2.0 (a.k.a. Swagger)
to Dhall.
```